src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -283,6 +302,7 @@ function getDrupalContent(buildOptions) {
       log('Failed to pipe Drupal content into Metalsmith!');
       if (
         buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST ||
+        buildOptions[USE_CMS_EXPORT_BUILD_ARG] ||
```

Comment:
So, this means that we will show an error if we fail and `--use-cms-export` was passed?

Reply:
Right, exactly. Plugins that follow this one will require the output of this one, so it doesn't really make sense to try plowing on through the content build when we have no content.

Probably we can just always fail the step when this fails, but I didn't examine the ramifications of such a change. I just know the rest of the build failed during a `yarn build:content --use-cms-export` when this step did because `data` was undefined when another plugin required it. So rather than wait for it to fail later, I preferred failing quickly so the error from the `process-cms-exports` JSON schema validation / transformers weren't hidden.

Reply:
Cool, that makes sense.